apps/frontend/src/libs/components/table/styles.module.css

```diff
@@ -1,3 +1,10 @@
+.table-container {
+	width: 100%;
+	overflow: hidden;
```

Comment:
Do we need overflow hidden on container if we already have scroll inside?

Reply:
Yeah, it is for styling, because table has straight corners now, and some other things are getting out of borders. It's either `overflow: hidden` here, or a lot of additional styles everywhere else.